docs/source/conf.py

```diff
@@ -15,8 +15,11 @@
 import os
 import sys
 import re
+
 sys.path.insert(0, os.path.abspath('../../Wrappers/Python/'))
 
+from cil import version

```

Comment:
I don't think this exists unless you built/installed the package, which is probably your case?

Reply:
yes, we install cil so this exists